### ISSUE TITLE
Cannot load Radiance in LispWorks 7.1.1 on MacOS Mojave 10.14.3

### ISSUE THREAD

**Issue by wdhurley** [Shirakumo/radiance#35]
[20190311. Listener-Radiance.txt](https://github.com/Shirakumo/radiance/files/2952813/20190311.Listener-Radiance.txt)

**Comment by Shinmera** [comment]
Does LispWorks define a "DB" or "DATABASE" package?

**Comment by wdhurley** [comment]
No. Closest to "DB" or "DATABASE" is a debugging package "dbg". Thank you for your efforts. Radiance looks to be terrific, and I very much want to use it with LispWorks.

**Comment by Shinmera** [comment]
I don't know then. The error makes no sense to me. It looked like it might be conflating the symbols from multiple packages, hence the nonsense error about class precedence, but if that's not the case then I have no idea at the moment.

**Comment by wdhurley** [comment]
I'll keep trying, because I think the problem is in LispWorks, not Radiance. Thank you for the effort.

**Comment by wdhurley** [comment]
This might be it:
An explanation of why Clojure doesn't implement CLOS provides the following CLOS example:

(defclass a () ())
(defclass b () ())
(defclass c (a b) ())
(defclass d (b a) ())
(defclass e (c d) ())

(make-instance 'e)

Error: Error during finalization of class #<STANDARD-CLASS E
216B409B>: Cannot compute class precedence list for class: #<STANDARD-
CLASS E 216B409B>

Note: LispWorks 7.1.1 (current) gets the same result.

The article: "Rich Already Answered That!" under "Clojure Design Decisions"
https://gist.github.com/reborg/dc8b0c96c397a56668905e2767fd697f

**Comment by Shinmera** [comment]
That example is an uncomputable class precedence list, which is an error on all conforming implementations.

Obviously Radiance works fine on other implementations, so it has nothing to do with that.
